src/platform/nxp/common/ble_zephyr/BLEManagerImpl.cpp

```diff
@@ -597,7 +597,7 @@ CHIP_ERROR BLEManagerImpl::HandleGAPDisconnect(const ChipDeviceEvent * event)
     // Force a reconfiguration of advertising in case we switched to non-connectable mode when
     // the BLE connection was established.
     mFlags.Set(Flags::kAdvertisingRefreshNeeded);
-    PlatformMgr().ScheduleWork(DriveBLEState, 0);
+    TEMPORARY_RETURN_IGNORED PlatformMgr().ScheduleWork(DriveBLEState, 0);
```

Comment:
The next line is return CHIP_NO_ERROR.
It should just be simplified to 
```suggestion
    return PlatformMgr().ScheduleWork(DriveBLEState, 0);
```
so we return the failure to the caller if there one.